test/Integration/IntegrationTestBase.cs

```diff
@@ -432,6 +421,26 @@ public void Dispose()
             GC.SuppressFinalize(this);
         }
 
+        /// <summary>
+        /// Disposes all the running function hosts
+        /// </summary>
+        protected void DisposeFunctionHosts()
+        {
+            foreach (Process functionHost in this.FunctionHostList)
+            {
+                try
+                {
+                    functionHost.Kill();
+                    functionHost.Dispose();
+                }
+                catch (Exception e2)
+                {
+                    this.LogOutput($"Failed to stop function host, Error: {e2.Message}");
```

Comment:
@chlafreniere Great example where using the suggestion feature is useful 😄 
```suggestion
                catch (Exception ex)
                {
                    this.LogOutput($"Failed to stop function host, Error: {ex.Message}");
```